packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -0,0 +1,132 @@
+:local {
+  .checkbox {
+    color: var(--terra-form-checkbox-font-color, rgb(0, 0, 0));
+    display: block;
+    font-family: inherit;
+    margin-right: var(--terra-form-checkbox-container-margin-right, 2em);
+    margin-top: var(--terra-form-checkbox-container-margin-top, 0.625em);
+    
+    /* Hide actual input element */
+    .hidden-input {
```

Comment:
On second thought, I *should* be able to hide the custom checkbox and show the actual input when using the default theme, instead of always using the custom css checkbox across themes. This would stop me from trying to recreate the windows checkbox style for some themes. Most of the themeable variables would be set to none. I'll give another update after I've tinkered with this.

Reply:
^^ I'll close this pr for now and open another after this update.